### PR TITLE
[5.0.1] Profiletype-add-attribute

### PR DESCRIPTION
**Pull request by fedelocci**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/ui/profile-types/common/ProfileTypeForm.js [hunk 1]
```diff
@@ -126,7 +127,7 @@ export class ProfileTypeFormBody extends Component {
         <Row>
           <Col xs={12}>
             <Button
-              className="pull-right ProfileTypeFormBody__save--btn"
+              className="pull-right ProfileTypeForm__save-btn"
```
Comment (alessioscalici) [inline src/ui/profile-types/common/ProfileTypeForm.js]: This is a good fix for BEM. Does this impact any SASS selector?

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**